runtime_builders/publish_runtime.py

```diff
@@ -28,23 +29,45 @@
 def main():
     logging.getLogger().setLevel(logging.INFO)
     parser = argparse.ArgumentParser()
-    parser.add_argument('--infile', '-i',
+    parser.add_argument('--directory', '-d',
                         help='templated cloudbuild config file.',
                         required=True)
     parser.add_argument('--bucket', '-b',
                         help='GCS bucket to publish runtime to',
                         default='runtime-builders')
-    parser.add_argument('--builder-name',
-                        help='the name of the runtime or project '
-                        'associated with this builder',
-                        required=True)
     args = parser.parse_args()
 
-    templated_file_contents = _resolve_tags(args.infile)
-    logging.info(templated_file_contents)
-    logging.info(_publish_to_gcs(templated_file_contents,
-                                 args.builder_name,
-                                 args.bucket))
+    return _resolve(args.directory, args.bucket)
+
+
+def _resolve(directory, bucket):
+    try:
+        gcs_paths = []
+        for filename in os.listdir(directory):
+            filepath = os.path.join(directory, filename)
+            if filepath.endswith('.json'):
+                with open(filepath, 'r') as f:
+                    project_cfg = json.load(f)
+                    project_name = project_cfg['project']
+                    for builder in project_cfg['builders']:
+                        cfg = os.path.abspath(str(builder['path']))
+                        name = builder.get('name', 'default')
```

Comment:
Do you have the JSON format documented somewhere? I can't remember if we decided to drop 'default'.